modules/oadp-release-notes-1-3-4.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes-1-3.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oadp-release-notes-1-3-4_{context}"]
+= {oadp-short} 1.3.4 release notes
+
+The {oadp-first} 1.3.4 release notes list resolved issues and known issues.
+
+[id="resolved-issues-1-3-4_{context}"]
+== Resolved issues
+
+.The backup spec.resourcepolicy.kind parameter is now case insensitive
+
+Previously, the `backup spec.resourcepolicy.kind` parameter was only supported with a lower-level string. With this fix, it is now case insensitive. link:https://issues.redhat.com/browse/OADP-2944[{oadp-short}-2944]
+
+.olm.maxOpenShiftVersion prevents cluster upgrade to OCP 4.16 version
+
+The cluster `operator-lifecycle-manager` operator must not be upgraded between minor {OCP} versions. Use the `olm.maxOpenShiftVersion` parameter to prevent upgrading to {OCP} 4.16 version when {oadp-short} 1.3 is installed. 
+To upgrade to {OCP} 4.16 version, upgrade {oadp-short} 1.3 on OCP 4.15 version to {oadp-short} 1.4. link:https://issues.redhat.com/browse/OADP-4803[{oadp-short}-4803]
```

Comment:
```suggestion
.Use olm.maxOpenShiftVersion to prevent cluster upgrade to OCP 4.16 version

The cluster `operator-lifecycle-manager` operator must not be upgraded between minor {OCP} versions. Use of the `olm.maxOpenShiftVersion` parameter prevents upgrading to {OCP} 4.16 version when {oadp-short} 1.3 is installed. 
To upgrade to {OCP} 4.16 version, upgrade {oadp-short} 1.3 on OCP 4.15 version to {oadp-short} 1.4. link:https://issues.redhat.com/browse/OADP-4803[{oadp-short}-4803]
```

I'm not confident my suggestion is necessarily better, but the intention of this change is to block the upgrade from 4.15 to 4.16 when OADP 1.3 is installed.

Reply:
fixed, thanks